repository/a.json

```diff
@@ -665,6 +665,17 @@
 			]
 		},
 		{
+			"name": "Amxx Pawn",
+			"details": "https://github.com/evandrocoan/SublimeAmxxPawn",
+			"labels": ["language syntax", "amx mod x", "amxx pawn"],
```

Comment:
My bad, it seems pretty redundant. I removed it on 2833e30

But the amxmodx I think it is worth because when you look for `amxmodx` on the search you find absolutely nothing about it. Then after adding this package, you would find a syntax for its language:

1. https://packagecontrol.io/search/amx%20mod%20x

Amxmodx is a virtual machine which executes code written in pawn with the amx mod x API:

1. https://www.amxmodx.org/api/